operator/src/lib.rs

```diff
@@ -287,11 +237,22 @@ impl NifiState {
                             node_name,
                             &pod_name,
                             &cm_name,
-                            &version,
                             labels,
                         )?;
 
                         self.context.client.create(&pod).await?;
+
+                        if let Some(config) =
```

Comment:
More comments please, what's happening here?
It might be clear to you today but in a few weeks?